pwem/tests/data/test_data.py

```diff
@@ -58,7 +58,7 @@ def createDummyProtocol(projName):
     try:
         proj.launchProtocol(prot)
     except Exception as e:
-        print(str(e))
+        logger.error("Can't launch EMProtocol")
```

Comment:
Wouldn't it be convenient to print the error too?

Reply:
Done